velox/docs/functions/uuid.rst

```diff
@@ -0,0 +1,7 @@
+=============================
+UUID Functions
+=============================
+
+.. function:: uuid() -> uuid
```

Comment:
Don't you need to update velox/docs/functions.rst as well? Would you generate the docs and make sure they look ok?

Reply:
Hi @mbasmanova , how to generate the docs? can you give me a example or doc?

Reply:
See instructions in the Building section in https://github.com/facebookincubator/velox/blob/main/velox/docs/README.md

Reply:
@mbasmanova, I will take a look, thanks

Reply:
Updated velox/docs/functions.rst. And should the html files in branch gh-pages be updated in a separate PR?

Reply:
Thanks. There is a nightly CI job that will update gh-pages after PR lands.